packages/augur-sdk/src/state/db/DB.ts

```diff
@@ -222,6 +222,7 @@ export class DB {
       );
     }
 
+    console.log(`Syncing generic log DBs`);
```

Comment:
Should probably put this just before populating the lists, since errors can pop up in early promises while the others are still being invoked. Same with the other message below.